roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -57,6 +65,14 @@ def subclasses(cls):
             for subclass in subclass.subclasses():
                 yield subclass
 
+    @staticmethod
```

Comment:
Feels like it would be slightly cleaner with:

```
     @classmethod
     def load_checks(cls):
         """Dynamically import all check modules for the side effect of registering checks."""
         for name in os.listdir(os.path.dirname(__file__)):
             if name.endswith(".py") and name not in cls.LOADER_EXCLUDES:
                 import_module(__package__ + "." + name[:-3])
```